### PR TITLE
[wasm] Add batchnorm kernel.

### PR DESCRIPTION
**Pull request by annxingyuan**
To see the logs from the Cloud Build CI, please join either our [discussion](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs) or [announcement](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs-announce) mailing list.

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/2264)

### CODE REVIEW

**Comment by annxingyuan** [comment]
@dsmilkov @nsthorat 

I remember one of you showed me a page of TensorFlow ops that you said we should use as the reference for naming our kernels - could you remind of that link?

**Comment by nsthorat** [comment]
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/ops/ops.pbtxt

**Comment by nsthorat** [comment]
(wait for daniel LGTM before submitting)
